### DISCUSSION TITLE
Question - issue 32 Average Mach Speed

### DISCUSSION THREAD

**Discussion by hamzabatr** [Estia-advanced-programming/pandora-public#79] [Q&A]
# What is you question about : 

What is mach ?
How can we calculate the average mach speed ?

- Have you check if a similar questions exists in the issues   ? 
- Have you check the update page on the wiki    ?  https://github.com/Estia-advanced-programming/pandora-public/wiki/Update

**Comment by dhmmasson** [comment]
* Have you check if a similar questions exists in the issues ?
See #28

**Comment by dhmmasson** [comment]
[See also](https://github.com/Estia-advanced-programming/pandora-public/wiki/Pandora-%C2%BB-Instructions#constant-values)

It a simple matter of changing units
